### PR TITLE
doc: elaborate on release notes wrt netmasks

### PR DESCRIPTION
**Pull request by vasild**
doc: elaborate on release notes wrt netmasks

A minor followup from https://github.com/bitcoin/bitcoin/pull/19628#issuecomment-679958713.

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -141,7 +141,8 @@ Updated settings
 
 - Netmasks that contain 1-bits after 0-bits (the 1-bits are not contiguous on
   the left side, e.g. 255.0.255.255) are no longer accepted. They are invalid
-  according to RFC 4632.
+  according to RFC 4632. Netmasks are used in the `-rpcallowip` parameter and
```
Comment (jonatack) [inline doc/release-notes.md:144]: - perhaps s/parameter/option|configuration option/ (`Usage:  bitcoind [options]`)

- per `bitcoind -help-debug | grep -C3 -i cidr` and `git grep -ni netmask`, maybe verify if netmasks are also used for the `-whitelist` option

Comment (vasild) [reply doc/release-notes.md]: Updated, thanks!

**Review by jonatack — COMMENTED** [review]
LGTM, thanks for the changes per `git diff 04921cd 8de51d1`

**Comment by laanwj** [comment]
ACK https://github.com/bitcoin/bitcoin/pull/19802/commits/8de51d1513ada6d96b80f164da48088e5cdcbe3f
The travis lint run error seems like soms kind of unrelated network corruption issue, but running it again just in case.

**Review by theStack — APPROVED** [review]
ACK 8de51d1513ada6d96b80f164da48088e5cdcb
Checked where in the code subnet parsing happens via `git grep LookupSubNet`.

**Comment by practicalswift** [comment]
ACK 8de51d1513ada6d96b80f164da48088e5cdcbe3f

**Pull request merged**